Misc/NEWS.d/next/Library/2020-01-01-18-44-52.bpo-38871.3EEOLg.rst

```diff
@@ -0,0 +1 @@
+Fix :mod:`lib2to3` to be able to use in edge cases when handling filter
```

Comment:
This is what users will se in the changelog so please describe *what* ege cases didn't work and now do :)